monkey/monkey_island/cc/ui/src/components/report-components/SecurityReport.js

```diff
@@ -503,29 +285,62 @@ class ReportPageComponent extends AuthComponent {
   generateReportFooter() {
     return (
       <div id='footer' className='text-center' style={{ marginTop: '20px' }}>
-        For questions, suggestions or any other feedback
-        contact: <a href='mailto://labs@guardicore.com' className='no-print'>labs@guardicore.com</a>
-        <div className='force-print' style={{ display: 'none' }}>labs@guardicore.com</div>
+        For questions, suggestions, or any other feedback,
+        contact <a href='mailto://labs@guardicore.com' className='no-print'>labs@guardicore.com</a>
+        <div className='force-print' style={{ display: 'none' }}>labs@guardicore.com</div>.
```

Comment:
This probably should be changed to `support@infectionmonkey.com`, but let's not cram more changes into this PR. Typos in the report have nothing to do with plugins.

Reply:
Right, I'll open another PR with these changes.

Reply:
https://github.com/guardicore/monkey/pull/2899